tests/loadtests/appfwSDL/VulkanAppSDL/VulkanAppSDL.cpp

```diff
@@ -713,7 +713,7 @@ VulkanAppSDL::createDevice()
     err = vkctx.gpu.enumerateDeviceExtensionProperties(
                                                NULL,
                                                &deviceExtensionCount,
-                                               NULL);
+                                               (vk::ExtensionProperties*)NULL);
```

Comment:
Why not use `nullptr` here like you did at line 601?

Reply:
FIxed!
I guess I oriented myself on two line above, but the whole file is a mixture of NULL vs. nullptr.

Reply:
Yeah. It started out as all C code. I started converting to C++ (vulkan.hpp) but I've never had time to finish the job, I still hope to some day.